docker-compose.yml

```diff
@@ -819,6 +827,55 @@ services:
   #     - SMARTHOST_HOST=smtp
   #     - SMARTHOST_PORT=587
 
+  xtrabackup:
+    image: percona/percona-xtrabackup:8.0.33
+    container_name: xtrabackup
+    command: sleep infinity
+    volumes:
+      - backup-data:/backups:rw
+      - appwrite-mariadb:/var/lib/mysql:r
+    networks:
+      - appwrite
+    user: 'root:root'
+
+  appwrite-backup:
+    image: appwrite-dev
+    <<: *x-logging
+    entrypoint: db-backup
+    command:
+      - --database=db_fra1_03 #todo remove this!
```

Comment:
Can we remove it now? If not, when can we?

Reply:
Removed 👍